### PR TITLE
feat(rename): rename collection can be a function

### PR DESCRIPTION
**Pull request by jbarreau**

### CODE REVIEW

**Comment by jbarreau** [comment]
The related documentation : https://github.com/ForestAdmin/developer-guide/pull/240

File: src/_example/django/Dockerfile [hunk 1]
```diff
@@ -28,13 +28,14 @@ RUN \
...
-WORKDIR /app/src/_example/django/
 
 RUN curl -sSL https://install.python-poetry.org | python3 - ;\
     export PATH="/opt/poetry/bin:$PATH"
 
+COPY ./ /app/
+
+WORKDIR /app/src/_example/django/
+
 RUN /opt/poetry/bin/poetry install --no-cache
```
Comment (jeffladiray) [inline src/_example/django/Dockerfile:39]: Mh, not a fan of having this inside of this contribution

Comment (jeffladiray) [reply src/_example/django/Dockerfile:39]: (Same goes for non-related modification in _example)

**Review by jeffladiray — APPROVED** [review]
Both comments are most likely related to the age of the PR.

Consider them if you feel like.

**Pull request merged**